Add "Search RubyGems" extension

aleks opened the pull request:
## Description

Hello 👋 

I became a huge fan of Raycast recently but was a bit annoyed that i couldn't use it to look up information from [rubygems.org](https://rubygems.org/) - here is the fix for that! 🥳 

- Use **Search RubyGems** to quickly find gems you're looking for
- Use the **Copy gem string** command to copy Gemfile compatible strings (i.e. `gem 'rails', '6.1.4.1'`)
- Gain quick access to gem specific links like **documentation**, **bug trackers** and **source code**
- See gem **dependencies** directly in Raycast

Ruby-familiar developers should quickly understand what's going on, but here are a few screenshots to make it clear:

![light-dark](https://user-images.githubusercontent.com/86240/145280394-bc152e4e-ecb2-42d7-bea3-07802bbccb00.png)
![gem-options](https://user-images.githubusercontent.com/86240/145280414-61a6a13e-a4c7-4e6f-ac4b-826059cee6d1.png)
![gem-info](https://user-images.githubusercontent.com/86240/145280422-201650d8-5c2c-4dda-985a-597de1daf1b2.png)

## Type of change
- New extension

## Screencast
https://user-images.githubusercontent.com/86240/145280928-c10f4805-3510-4587-95ec-b9d36c7ee577.mov

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

pernielsentikaer commented on extensions/search-rubygems/src/components/GemOptions.tsx:
```diff
@@ -0,0 +1,62 @@
+          accessoryTitle="Dependencies and more"
+          actions={
+            <ActionPanel>
+              <PushAction
```
No icon added, please add an icon for PushAction

aleks replied:
@pernielsentikaer fixed!

pernielsentikaer commented on extensions/search-rubygems/src/index.tsx:
```diff
@@ -0,0 +1,59 @@
+            subtitle={gem.info}
+            actions={
+              <ActionPanel>
+                <PushAction
```
No icon added, please add an icon for PushAction

aleks replied:
@pernielsentikaer fixed!

pernielsentikaer reviewed (changes requested):
Hi @aleks ⭐ 

Thanks for your contribution, I left a few minor corrections that should be addressed 💪 

Looking forward to test it out again 💯 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

aleks commented:
@pernielsentikaer thanks for the review! 🥳 I added icons on both `PushAction`s and updated the titles a bit. Let me know if you need anything else! 🤘

pernielsentikaer commented on extensions/search-rubygems/package.json:
```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "search-rubygems",
+  "title": "Search RubyGems",
```
```suggestion
  "title": "Search Documentation",
  "subtitle": "RubyGems",
```

pernielsentikaer reviewed (changes requested):
Hi @aleks 👋 

Just found a small one, could you correct this, when it's done I think we're ready to merge 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

pernielsentikaer reviewed (approved):
Hi @aleks 🔥 

Thanks for the update! I'm going to accept it and then will my colleague do a final review before it's finally merged 🚀
